sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1233,8 +1218,9 @@ static int create_sdw_dailink(struct snd_soc_card *card,
 	codec_index = find_codec_info_part(link->adr_d[adr_index].adr);
 	if (codec_index < 0)
 		return codec_index;
+	codec_info = &codec_info_list[codec_index];
 
-	if (codec_info_list[codec_index].ignore_pch_dmic)
+	if (codec_info->ignore_pch_dmic)
```

Comment:
i'd also split this par about adding codec_info into a separate patch first so the gist of this patch is easy to see

Reply:
> i'd also split this par about adding codec_info into a separate patch first so the gist of this patch is easy to see

ack